clair/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for clair.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+logLevel: info
+insecureTls: false
+image:
+  repository: quay.io/coreos/clair-git
+  tag: latest
+  pullPolicy: Always
+service:
+  type: NodePort
+  internalApiPort: 6060
+  externalApiPort: 6060
+  internalHealthPort: 6061
+  externalHealthPort: 6061
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+resources:
+  limits:
+    cpu: 200m
+    memory: 1500Mi
+  requests:
+    cpu: 100m
+    memory: 500Mi
+config:
+  # postgresURI: "postgres://user:password@host:5432/postgres?sslmode=disable"
+  paginationKey: "XxoPtCUzrUv4JV5dS+yQ+MdW7yLEJnRMwigVY/bpgtQ="
```

Comment:
You *must* remove this key and force the users to handle this. My understanding is that you have to generate a key if you want federation, otherwise it will automatically random created.